src/gas/types.rs

```diff
@@ -269,7 +269,7 @@ fn _hook_init() {
 			&mut vec![data.get(data.get(i)?)?],
 		)?;
 	}
-	REACTION_INFO.write().insert(get_reaction_info());
+	let _ = REACTION_INFO.write().insert(get_reaction_info());
```

Comment:
This is actually fixed on master already by making it idiomatic (i.e. using = instead of write(), as the warning suggests), so probably just remove them

Reply:
done